### PR TITLE
Config tfa front

### PR DESCRIPTION
**Pull request by vfurmane**
Need to merge:
- #52 
- #57 
- #59 
- #60 
- #61

### CODE REVIEW

**Comment by vfurmane** [comment]
Oops, need to merge #53 first

**Review by deblokc — APPROVED** [review]

File: apps/web/components/ProfilePage/ChangePswrd.tsx [hunk 1]
```diff
@@ -1,25 +1,111 @@
...
+  old_password: string;
+  new_password: string;
+  confirm_new_password: string;
+}
+
+async function changePassword(
+  data: ChangePasswordFormData
+): Promise<{ message: string } | null> {
+  const response = await fetch(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL}/auth/change_password`,
```
Comment (maxenceeudier) [inline apps/web/components/ProfilePage/ChangePswrd.tsx]: we use /api now for the fetch methode.

File: apps/web/components/ProfilePage/ConfigTfa/ManageTfa.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import { Button } from "../../Button";
+
+export interface ManageTfaProps {
+  success: () => void;
+}
+
+async function deleteTfa(): Promise<void | null> {
+  const response = await fetch(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL}/auth/tfa`,
+    {
```
Comment (maxenceeudier) [inline apps/web/components/ProfilePage/ConfigTfa/ManageTfa.tsx:10]: same for /api

File: apps/web/components/ProfilePage/ConfigTfa/SetupTfa.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { Button } from "../../Button";
+import { Dispatch, SetStateAction } from "react";
+
+async function setupTfa(): Promise<{ otpauth_url: string } | null> {
+  const response = await fetch(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL}/auth/tfa`,
```
Comment (maxenceeudier) [inline apps/web/components/ProfilePage/ConfigTfa/SetupTfa.tsx:6]: /api

**Review by maxenceeudier — APPROVED** [review]

**Review by savosaicic — APPROVED** [review]

**Review by soleraeus — APPROVED** [review]

**Pull request merged**